lib/engine/game/g_1868_wy/step/manual_close_company.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+
+module Engine
+  module Game
+    module G1868WY
+      module Step
+        class ManualCloseCompany < Engine::Step::Base
+          ACTIONS = %w[manual_close_company].freeze
+
+          def description
+            'Manually Close Companies'
+          end
+
+          def actions(entity)
+            return [] unless entity.company? &&
```

Comment:
don't use unless with multiple conditions

Reply:
Hopefully this is the last time you'll need to make this comment that lol #8564